core/src/MsalIdHelper.cs

```diff
@@ -35,83 +35,103 @@ namespace Microsoft.Identity.Core
     internal static class MsalIdParameter
     {
         /// <summary>
-        /// MSAL Flavor: .NET or WinRT
+        ///     MSAL Flavor: .NET or WinRT
```

Comment:
>      [](start = 11, length = 5)

not sure I understand the new indents we are getting... I'm pretty sure my VS is not setup to do just that

Reply:
R#